attributes/default.rb

```diff
@@ -426,6 +426,7 @@
 
 #git
 default['hopsworks']['git_command_timeout_minutes']  = "60"
+default['hopsworks']['read_only_git_repositories']  = "false"
```

Comment:
Can you rename the variable to be called `enable_read_only_git_repositories`? We want to have the feature flags prefixed with `enable_`